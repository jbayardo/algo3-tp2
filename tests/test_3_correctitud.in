0 1 60; 1 2 58; 1 3 30; 0 3 20; 0 2 18; 2 3 15
0 1 15; 1 2 30; 1 3 46; 2 3 100; 0 3 50; 0 2 90
0 1 10; 1 2 16; 1 3 14; 1 4 12; 0 2 20; 2 3 25; 3 4 22; 0 4 9
0 1 25; 1 2 18; 1 3 20; 1 4 22; 0 2 10; 2 3 12; 3 4 13; 0 4 16
0 1 8; 1 2 7; 2 3 6; 0 3 9; 0 2 10
0 1 2; 1 2 4; 2 0 1
0 1 10; 1 2 6; 2 3 12; 0 3 5; 0 4 14; 3 4  14; 2 5 7; 1 5 8; 4 5 16