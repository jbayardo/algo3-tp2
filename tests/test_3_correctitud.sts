Excercise 3 Timer			22,19,25,24,15,10,28
