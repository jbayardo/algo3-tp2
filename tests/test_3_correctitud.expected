53
95
45 
51
13
0
26
